rfcs/045-sierra-work-relationships/README.md

```diff
@@ -0,0 +1,377 @@
+
+In the API, this will manifest in the `parts` list.
+
+This extract from the [Basil Hood Photograph Album](https://search.wellcomelibrary.org/iii/encore/record/C__Rb1172977__Sbasil%20hood__P0,1__Orightresult__U__X6?lang=eng&suite=cobalt&marcData=Y)
+```
+...
+774 0  |gPage 6 :|tCharing Cross Hospital: a portrait of house 
+       surgeons. Photograph, 1906.|w(Wcat)28916i 
+774 0  |gPage 7 :|tCharing Cross Hospital: a portrait of house 
+       surgeons. Photograph, 1906.|w(Wcat)28922i 
+774 0  |gPage 7 :|tCharing Cross Hospital: a portrait of house 
+       surgeons. Photograph, 1906.|w(Wcat)28924i 
+...
+```
+becomes
+```
+  "parts":[
+    ...
+    {
+      "id": "[canonical id derived from](Wcat)28916i"
+      "subPart": "Page 6"
+      "title": "Charing Cross Hospital: a portrait of house surgeons. Photograph, 1906."
+    },
+    {
+      "id": "[canonical id derived from](Wcat)28922i"
+      "subPart": "Page 7"
+      "title": "Charing Cross Hospital: a portrait of house surgeons. Photograph, 1906."
+    },
+    {
+      "id": "[canonical id derived from](Wcat)28924i"
+      "subPart": "Page 7"
+      "title": "Charing Cross Hospital: a portrait of house surgeons. Photograph, 1906."
+    }
+    ...
+  ]
+```
+## Unanswered Questions
+
+### 774 without id
+
+Some 774 values have no id, e.g. [Catalogues of Malayan plants, birds and snakes](https://search.wellcomelibrary.org/iii/encore/record/C__Rb3017508__SLists%20of%20snake%20names%20in%20Malay__Orightresult__U__X3?lang=eng&suite=cobalt&marcData=Y)
+[Wellcome Malay 8](https://wellcomecollection.org/works/s5cmqb9q) on wellcomecollection.org.
+
+I don't know what that means. This cannot be presented as any kind of list
+of links in a UI, because there is nothing to link to.
+
+This example is a TEI manuscript, so the data for the API comes from there
+rather than Sierra.
+
+```
+245 00 [Catalogues of Malayan plants, birds and snakes] 
+246 1  MS Malay 8 
+
+...
+
+774 1  |tLists of plants collected in January 1885.|h30 p.; 34 x 
+       21 cm 
+774 1  |tLists of snake names in Malay.|h4 p.; 34 x 21 cm 
+774 1  |tLists of Malacca birds.|h6 p.; 34 x 21.5 cm 
+774 1  |tLists of varieties of rice and rattan.|h2 p.; 34 x 21 cm
+774 1  |tNames of plants.|h2 p.; 34.5 x 20 cm 
+774 1  |tLists of plant names and uses.|h3 p.; 34.5 x 21.5 cm 
+774 1  |tLists of grasses collected in Malacca.|h13 p.; 34 x 21 
+       cm 
+774 1  |tNames of plants collected at Mĕrliman.|h9 p.; 34 x 21 cm
+
+```
+
+### Other potential subPart subfields
+
+There are also subfields $p (Name of part/section of a work) and $n (Number of part/section of a work)
+Both of these are present on at least one example:
+
+[Madame Delait, the bearded woman of Plombières](https://search.wellcomelibrary.org/iii/encore/record/C__Rb1532371__Sfallaize__P0,2__Orightresult__U__X3?lang=eng&suite=cobalt&marcData=Y)
+
+However, in this case, the values do not look useful:
+
+`830  0 Fallaize Collection.|pname of grouping ;|nnumber`
```

Comment:
Yes. With these, I fancy that the right thing to do (at our end) is not worry about them yet, but bear in mind that they do exist and may hold useful information.

We can combine that with asking whether they do ever hold useful information, so that we can include it if we need to in the future.

In terms of linking, because this is (at best) another subPart, it falls under step 4 - display (or use, as your other comment suggests) the subPart information